server/app/views/admin/programs/ProgramBlocksView.java

```diff
@@ -796,7 +838,7 @@ private Optional<FormTag> optionalToggle(
   }
 
   // Creates label that can be used in the read only view, for example to replace a toggle.
```

Comment:
I know it's not a part of this PR but if you wouldn't mind changing this from a // comment to a javadoc comment, that would be great

Reply:
Sure.